.travis-ci.yml

```diff
@@ -0,0 +1,10 @@
+language: minimal
```

Comment:
Travis also offers `python` with first class support, which doesn't give you control over a very precise Python version but it allows [multiple builds](https://docs.travis-ci.com/user/languages/python/#specifying-python-versions) for libraries, like trying out Python 3.8 compatibility alongside 3.7. Opinions? I am not leaning either way.

Reply:
I like this in theory but I like to be able to run the same tests locally. Also, I was thinking about whether to test lots of version variations. Given this package only uses in-built Flask classes/functions, which is tested quite thoroughly, I copied the versions of python 3 that Flask specifies in its `setup.py` file and feel comfortable not testing framework code. Happy to add lots of variations if people have opinions on this.

Reply:
I agree local reproducibility of the environment is important, I can imagine other libraries where the same approach would involve installing a OS package for example. Let's subordinate this decision to the minimum Python version choice in `setup.py` which is what should drive this (e.g. if it absurdly supported 2.7, it would be tested with that)